File: c++/numeros_pares.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {

    int n, cont = 0;

    cout << "Quanto numeros voce vai digitar? ";
    cin >> n;

    int vet[n];

    for(int i = 0; i < n; i++) {
        cout << "Digite um numero: ";
        cin >> vet[i];
    }

    cout << endl << "Numeros pares:" << endl;
    for(int i = 0; i < n; i++) {
        if(vet[i] % 2 == 0) {
            cout << vet[i] << " ";
            cont = cont + 1;
        }
    }

    cout << endl << "\nQUANTIDADE DE PARES = " << cont << endl;

    return 0;
}

File: java/numeros_pares.java
import java.util.Locale;
import java.util.Scanner;

public class numeros_pares {

	public static void main(String[] args) {
		
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		int N, cont;
		
		System.out.print("Quantos numeros voce vai digitar? ");
		N = sc.nextInt();
		
		int[] vet = new int[N];
		
		for (int i = 0; i < N; i++) {
			System.out.print("Digite um numero: ");
			vet[i] = sc.nextInt();
		}
		
		System.out.println();
		System.out.print("NUMEROS PARES: ");
		for (int i = 0; i < N; i++) {
			if (vet[i] % 2 == 0) {
				System.out.print(vet[i] + " ");
			}
		}
		
		System.out.println();
		
		cont = 0;
		for (int i = 0; i < N; i++) {
			if (vet[i] % 2 == 0) {
				cont = cont + 1;
			}
		}
		System.out.println("QUANTIDADE DE PARES = " + cont);
		
		sc.close();
	}

}
